### PR TITLE
copr: add metrics

### PR DESCRIPTION
**Pull request by BusyJay**
@ngaut @siddontang PTAL

### CODE REVIEW

File: src/bin/tikv-server.rs [hunk 1]
```diff
@@ -371,16 +371,8 @@ fn main() {
                 "dsn: rocksdb, raftkv");
     opts.optopt("I", "cluster-id", "set cluster id", "must greater than 0.");
     opts.optopt("", "pd", "set pd address", "host:port");
-    opts.optopt("M",
-                "metric",
-                "set metric level",
-                "metric level: trace, debug, info, warn, error, off");
     opts.optopt("", "metric-host", "set statsd server address", "host:port");
```
Comment (siddontang) [inline src/bin/tikv-server.rs]: I prefer addr, host usually doesn't contain port.

File: src/server/coprocessor.rs [hunk 2]
```diff
@@ -199,12 +199,12 @@ impl TiDbEndPoint {
                 return;
             }
         };
-        debug!("[TIME_SNAPSHOT] {:?}", ts.elapsed());
+        metric_time!("COPR_SNAPSHOT", ts.elapsed());
```
Comment (siddontang) [inline src/server/coprocessor.rs]: I think we should use "copr" tag first like "raftstore", so here may be "corp.snapshot"

Comment (siddontang) [inline src/server/coprocessor.rs]: Btw, I suggest lower case for all metric keys, seem better read. 
@ngaut

**Comment by siddontang** [comment]
LGTM

**Comment by ngaut** [comment]
LGTM

**Pull request merged**